packages/storybook/src/stories/storybook.css

```diff
@@ -4,3 +4,12 @@
   text-align: center;
   font-size: small;
 }
+
+.cdc-react.table .checkbox-content {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  clip: rect(0, 0, 0, 0);
```

Comment:
I had to lookup `clip` since it's been a minute since I've used it and apparently its deprecated 🤷  https://developer.mozilla.org/en-US/docs/Web/CSS/clip

I looked at what the CMS Design System does for their "screen reader visibility only" utility class and they use the same so I doubt browsers will start dropping it anytime soon. https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/styles/_mixins.scss#L106

Plus `storybook.css` is only for supporting component demos though so no blocker here. We may want to have a `.visibility--screen-reader` global utility class at some point.

Reply:
We have a `sr-only` global class already. I decided to add a prop of `srOnly` to the checkbox component. This way the label is picked up by the screen reader, but we keep the table according to the design document. May you please re-review?